dvach-browser/VLCPlayerViewController.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  VLCPlayerViewController.swift
+//  dvach-browser
+//
+//  Created by Victor Cebanu on 2/3/20.
+//  Copyright © 2020 8of. All rights reserved.
+//
+
+import Foundation
+
+final class VLCPlayerViewController: UIViewController, VLCPlayerViewInput {
+    var interactor: VLCPlayerViewOutput!
+    var player: VLCMediaPlayer?
+
+    @IBOutlet weak var playerView: UIView!
```

Comment:
I suggest we mark everything `private` if we aren't accessing it from outside. Even `IBOutlet`s and `IBAction`s.